### DISCUSSION TITLE
Does `dmc` support virtual colors?

### DISCUSSION THREAD

**Discussion by mmarfat** [snehilvj/dash-mantine-components#402] [Q&A] [answered]
In [mantine.dev](https://mantine.dev/theming/colors/#virtual-colors) there's an option to set custom color pairs for light/dark theme via the `virtualColor()` function. For example, `cyan` for light mode and `pink` for dark mode.

Is this possible in `dmc`?

**Comment by AnnMarieW** [comment, marked as answer]
Hi @mmarfat 

Currently dmc does not support [functions as props](https://github.com/snehilvj/dash-mantine-components/issues/356) - but it's on the project plan for after the first major release.

However, this function is quite simple and we may be able to handle it with a regular dash prop, then calling the function inside the component.  It would take some investigating  but it would be a great addition to dmc!

**Reply by mmarfat** [reply]
I thought so, thanks for the quick response! I'll see if I can dig into it a bit myself when I get the time.
